<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EATC BANK MAIN PAGE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QPushButton" name="bank_op_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>BANK OPERATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainpagecutomeroperationbutton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>CUSTOMER OPERATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>EACT BANK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
